docs/content/en/blog/news/january-2022-update.md

```diff
@@ -0,0 +1,127 @@
+
+An application can contain one or more labels in its application configuration file as below:
+
+```yaml
+apiVersion: pipecd.dev/v1beta1
+kind: TerraformApp
+spec:
+  name: simple-app
+  envName: production
+  labels:
+    #env: production
+    service: backend
+  ...
+```
+
+Filtering application or deployment by the label is being implemeted currently. We hope that it will be shipped in the next release.
+
+##### Simplifying the way of registering application
+
+Before [v0.23.0](https://github.com/pipe-cd/pipe/releases/tag/v0.23.0), while registering a new application user has to configure the application information manually. But now most of the information can be specified in the application configuration file. And Piped agents automatically find the un-registered applications to suggest to users. It means that all you have to do is just a few clicks on the web console.
+
+![](/images/registering-an-application-from-git.png)
+<p style="text-align: center;">
+Picking from the suggested application list to register
+</p>
+
+Please note that to be suggested, the application configuration file must be suffixed by `.pipecd.yaml`. And the default name of that file has been changed to `app.pipecd.yaml` instead of `.pipe.yaml` as before.
+
+##### Showing the connection status of Pipeds
+
+The connection status of Piped agents to the control plane is shown on the Settings page in real-time. This gives the operator a quick look at the current status of a particular Piped agent is.
+
+![](/images/january-2022-piped-connection-status.png)
+
+##### More controls on triggering the deployment
+
+By default, when a new merged pull request touches an application, a new deployment for that application will be triggered to execute the sync process.
+
+From `v0.22.0`, a new [`trigger`](https://pipecd.dev/docs/user-guide/configuration-reference/#deploymenttrigger) field has been added to the application configuration file to allow users to customize the triggering condition. For example, using `onOutOfSync` to enable the ability to attempt to resolve `OUT_OF_SYNC` state whenever a configuration drift has been detected.
+
+The deployment triggering of each application can be configured with the following fields:
+
+* `onCommit`: controls triggering new deployment when newly added Git commits touched the application.
+* `onCommand`: controls triggering new deployment when received a new `SYNC` command from the web console or `pipectl`
+* `onOutOfSync`: controls triggering new deployment when application is at `OUT_OF_SYNC` state. Enabling this will force Piped to always attempt to keep the application as synced as possible.
+
```

Comment:
should we add `onChain` trigger description here as well, since it's released from `v0.23.0` already?

Reply:
Nice catch! Let me add it.

Reply:
Fixed!